test/e2e/ec2/instance.go

```diff
@@ -135,24 +138,12 @@ func DeleteEC2Instance(ctx context.Context, client *ec2.Client, instanceID strin
 }
 
 func newDefaultRunEC2Retrier() runInstanceRetrier {
-	return runInstanceRetrier{
-		Standard:   retry.NewStandard(),
-		maxRetries: 60,
-		backoff:    2 * time.Second,
-	}
-}
-
-type runInstanceRetrier struct {
-	*retry.Standard
-	maxRetries int
-	backoff    time.Duration
+	return runInstanceRetrier{}
 }
 
-func (c runInstanceRetrier) IsErrorRetryable(err error) bool {
-	if c.Standard.IsErrorRetryable(err) {
-		return true
-	}
+type runInstanceRetrier struct{}
```

Comment:
now that this is not a retrier, should we change the name?
like
```suggestion
type invalidInstanceProfileRetryable struct{}
```